app/test_main.py

```diff
@@ -1,3 +1,28 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_years_to_convert, dog_years_to_convert, expected_years",
+    [
+        (0, 0, [0, 0]),
```

Comment:
u shoud also chek the invalid input (under zero values) and rising error by checklist
![image](https://github.com/mate-academy/py-cat-and-dog-years/assets/154929153/4be791c5-b296-44bd-8c1d-01fa8d70a78f)